cmd/ddev/cmd/remove_test.go

```diff
@@ -54,3 +46,23 @@ func TestDevRemove(t *testing.T) {
 	err = addSites()
 	require.NoError(t, err)
 }
+
+// TestDdevRemoveDeletedDir tests that a project can be `ddev remove`d when its project directory has been removed
+func TestDdevRemoveDeletedDir(t *testing.T) {
```

Comment:
This PR (at the moment) only deals with collecting projects passed in as args, so it's kind of arg processing -- maybe one step removed. The test you're referring to is testing `app.Stop()`, which is important because it's wrapping a `docker-compose` command that needs project configuration files to get started, so it will return an error if the project directory is missing. The TestDdevStopMissingDirectory test is passing because the error is getting returned as expected. The `ddev remove` command will _attempt_ an `app.Stop()`, but when it returns an error because the project directory is missing, will emit a warning and continue to remove the containers without using `docker-compose`. 

That's not to say they shouldn't behave consistently, just explaining why it's like that now. Some commands operate on containers directly, some rely on `docker-compose`, I'll see if it needs to be that way.